### DISCUSSION TITLE
Error upload Reality Model

### DISCUSSION THREAD

**Discussion by ovaisa** [iTwin/itwinjs-core#1897] [Q&A]
I am getting the following error when using iTwin Synchronizer.

The reality model is in 3sm format and I can see it inside MicroStation.  The dgn file also has a CAD model and the reality mesh is attached using Attach>Reality Mesh tool.   

On Connected Project, I have Project Admin roles which essentially has all services and workflows enabled include PW Context Share - assign/create/manage/use.

Can you please guide what I might be doing wrong. 

Thanks!

![SynchronizerError](https://user-images.githubusercontent.com/83007711/126350343-aa5c56bc-12ff-49db-86e7-cb7f91441299.JPG)

**Comment by rbois-bentley** [comment]
Hi, we are sorry this has been too long without answer. Is this still an issue for you?
